build/ical.js

```diff
@@ -886,7 +894,9 @@ ICAL.design = (function() {
 
   // When adding a value here, be sure to add it to the parameter types!
   var vcardValues = ICAL.helpers.extend(commonValues, {
-
+    component: textType(FROM_ICAL_NEWLINE, TO_ICAL_NEWLINE, "text"),
```

Comment:
Yes `component` is always in a `strcuturedValue`. That is also a way to solve this, not sure which one is the better solution.